lib/org/chromium/webidl/LocalGitRunner.js

```diff
@@ -15,6 +15,8 @@ foam.CLASS({
     'org.chromium.webidl.URLExtractor',
   ],
 
+  imports: [ 'outputDAO?' ],
```

Comment:
I only intend on it being configured by the owner. I dont think there are any logical default values that we could set it to from within the components themselves.